index.md

```diff
@@ -175,22 +180,20 @@ sequenceDiagram
 
 {% include alert.html body='This is a standard alert.' %}
 
-{% include alert.html type='success'
-                      icon='check_circle'
-                      body='This is a successful alert.' %}
+{% include alert.html type='success' icon='check_circle'
+body='This is a successful alert.' %}
 
-{% include alert.html icon='info_outline'
-                      header='**Informational** *alert*'
-                      body='This is an **informational** alert *with* `<markdown/>`{:.language-html}.' %}
+{% include alert.html icon='info_outline' header='**Informational**     *alert*'
+body='This is an **informational**   alert *with*   `<markdown/>`{:.language-html}.'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)